### ISSUE TITLE
Linkouts for items per track?

### ISSUE THREAD

**Issue by MichielVanBel** [igvteam/igv.js#430]
In an annotation track, when clicking on a transcript, you currently get a small pop-up window with the name of the transcript and the associated strand.

Wouldn't it be useful here to make the value in the name-field a URL (to link back to an annotation platform for example), or alternatively provide a link-out field (so beneath the name/strand) that can contain this URL per gene? 

The configuration could be within the track-definition with a prefix-URL part, to which the transcript-id is then simply appended. Other systems might work fine as well of course.

**Comment by jrobinso** [comment]
Let me rephrase to see if I understand.    Add a track config option to specify a URL to add to a features popup detail window.   The URL could include a wildcard for substituting the feature name or other specified attribute.   For example

infoLink: "http://www.genecards.org/cgi-bin/carddisp.pl?gene=$FEATURE$"

**Comment by MichielVanBel** [comment]
Exactly something like that. It's just that, if you want to include IGV.js in a website, then some kind of interactive way of linking to/from other websites is a necessity.

**Comment by matthdsm** [comment]
+1 on this, would be useful for us too!

**Comment by rl1m** [comment]
+1, love this feature

**Comment by oleraj** [comment]
+1, would be very useful here too.  We'd be interested in using this for annotation tracks (e.g., genes) as well as variant tracks (e.g., linkout to lookup coordinates on external website).

**Comment by oleraj** [comment]
After a bit of digging, it looks like this is potentially already possible in the way described by this example:
https://github.com/igvteam/igv.js/blob/master/examples/events/custom-track-click.html
Does that example work to allow clicking out to an external site?

**Comment by jrobinso** [comment]
@oleraj The example that is most relevant, perhaps, is custom-track-popover, which you can run in the link below.  Note there is a link to an external site inserted into the poup.

http://igv.org/web/release/2.7.4/examples/events/custom-track-popover.html

**Comment by oleraj** [comment]
Thanks, @jrobinso, we'll try this out.

**Comment by jrobinso** [comment]
Hi all, sorry its taken so long to close this.   In addition to the customization above, I just pushed a configuration option to be released with 2.10.0 shortly, example follows.   $$ in the url is replaced with the feature name

```
{
  url: "https://s3.amazonaws.com/igv.broadinstitute.org/annotations/hg19/genes/gencode.v18.collapsed.bed.gz",
  indexURL: "https://s3.amazonaws.com/igv.broadinstitute.org/annotations/hg19/genes/gencode.v18.collapsed.bed.gz.tbi",
  name: "Gencode v18",
  infoURL: "https://www.ncbi.nlm.nih.gov/gene/?term=$$"
}
```

In addition, for bed files you can set this in a track line with the "url" property, per UCSC convention.  This has actually been supported for a number of releases, but 

```
track name=clones description="Clones" visibility=2 color=0,128,0 useScore=1 url="http://genome.ucsc.edu/goldenPath/help/clones.html#$$"
#chrom chromStart chromEnd name score
chr22 10000000 10004000 cloneA 960
chr22 10002000 10006000 cloneB 200
chr22 10005000 10009000 cloneC 700
chr22 10006000 10010000 cloneD 600
chr22 10011000 10015000 cloneE 300
chr22 10012000 10017000 cloneF 100
```
